src/metrics/mod.rs

```diff
@@ -203,16 +242,10 @@ macro_rules! request {
     }
 }
 
-heatmap!(
-    REQUEST_LATENCY,
-    "request_latency",
-    "distribution of request latencies in nanoseconds. incremented at time of requests disbatch."
-);
-
 heatmap!(
     RESPONSE_LATENCY,
     "response_latency",
-    "distribution of response latencies in nanoseconds. incremented at time of response receipt."
+    "distribution of response latencies in nanoseconds."
```

Comment:
And similar renaming here at the call sites.

Reply:
Fixed.